backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUPIItemProductDAO.java

```diff
@@ -291,10 +295,12 @@ public I_M_HU_PI_Item_Product retrieveMaterialItemProduct(
 			}
 		}
 
+		final boolean enforceHU_PI_Item_Product = sysConfigBL.getBooleanValue(SYSCONFIG_Enforce_M_HU_PI_Item_Product_ID, false);
+		
 		//
 		// Product Price of a Price List Version has this Packing Item
 		{
-			if (queryVO.getPriceListVersionId() != null)
+			if (enforceHU_PI_Item_Product && queryVO.getPriceListVersionId() != null)
```

Comment:
do this checking in de.metas.ui.web.quickinput.field.PackingItemProductFieldHelper and decide there if u set queryVO's priceListVersionId.